serving/docker/partition/sm_neo_neuron_partition.py

```diff
@@ -432,37 +281,49 @@ def copy_input_files_to_output(self):
                 if os.path.abspath(entry.path) != optimized_model_dir:
                     shutil.move(entry.path, optimized_model_dir)
 
+        # use safetensors index to determine which weights to skip copying
+        ignore_fun = None
+        try:
+            safetensors_index_file = os.path.join(
+                self.INPUT_MODEL_DIRECTORY, "model.safetensors.index.json")
+            # build the set of weight files to ignore
+            weight_files = set()
+            with open(safetensors_index_file) as f:
+                index = json.load(f)
+                for k, v in index["weight_map"].items():
+                    weight_files.add(v)
```

Comment:
The index file itself can be exempt from copying since the weights mentioned in the index file was not copied.

Reply:
Done
https://github.com/deepjavalibrary/djl-serving/blob/2f131026b30d1c4ea72641b395e2a1b0b5341b14/serving/docker/partition/sm_neo_neuron_partition.py#L291